google-ads/pom.xml

```diff
@@ -33,10 +33,10 @@
   </description>
 
   <properties>
-    <auto-value.version>1.4</auto-value.version>
+    <auto-value.version>1.6.6</auto-value.version>
     <gax.version>1.50.1</gax.version>
     <grpc.version>1.25.0</grpc.version>
-    <protobuf.version>3.5.1</protobuf.version>
+    <protobuf.version>3.10.0</protobuf.version>
```

Comment:
Can we remove these dependencies and defer to the version requested by GAX?

Reply:
_Almost_, with the exception of the transitive dependency in tests on `grpc-context`. See additional `test` scope dependency below.

I'm keeping these properties because putting them all side-by-side with the netty version will make it easier to verify consistency going forward.